src/EventListener/ReadRateLimitAnnotationListener.php

```diff
@@ -78,7 +78,9 @@ public function onKernelController(ControllerEvent $event): void
                 $annotation->getPeriod() ?? $this->period
             );
 
-            $rateLimit->varyHashOn('_route', $request->attributes->get('_route'));
+            /** @var string */
```

Comment:
```suggestion
            /** @var string $route */
```
Non-blocking (the same applies to different modifications in the PR)

Reply:
I tried, but then CI fails and cs-fix remove some of them and the PR with rector fix / remove others

Reply:
Changed